api/database/sampleApp.py

```diff
@@ -0,0 +1,68 @@
+from flask import Flask, request, jsonify
+from mongoengine import *
+from pymongo.errors import DuplicateKeyError
+from users import Users
+
+app = Flask(__name__)
+
+@app.route('/users', methods = ["GET"])
+def get_users():
+    name = request.args.get('Name')
+    email = request.args.get('Email')
+    password = request.args.get('Password')
+
+    data = []
+    for user in Users.objects:
+        if name is not None:
+            if user.name == name:
+                data.append(user.to_json())
+            else:
+                pass
```

Comment:
We don't need to specify passing since that's what happens by default I believe.